### PR TITLE
GH-84

### PR DESCRIPTION
**Pull request by evdhoudt**
Enhances NCI-Agency/anet#84

### CODE REVIEW

File: client/src/index.css [hunk 1]
```diff
@@ -626,6 +626,28 @@ header.searchPagination {
 	margin-bottom: 0;	
 	border-radius: 0;
 	border: 1px solid transparent;
+	animation: flash 1.25s forwards linear normal;
+}
```
Comment (VassilIordanov) [inline client/src/index.css]: can we add
	animation-iteration-count: 5;

File: client/src/index.css [hunk 2]
```diff
@@ -626,6 +626,28 @@ header.searchPagination {
...
+.general-banner.alert-warning {
+	background-color: #fce469;
+}
+.general-banner.alert-danger {
+	background-color: #f22a2a;
+	color: #fff;
+}
+
+@keyframes flash {
+	5% {
```
Comment (VassilIordanov) [inline client/src/index.css]: and change 5% to 50%

File: client/src/index.css [hunk 3]
```diff
@@ -626,6 +626,28 @@ header.searchPagination {
 	margin-bottom: 0;	
 	border-radius: 0;
 	border: 1px solid transparent;
+	animation: flash 1.25s forwards linear normal;
```
Comment (VassilIordanov) [inline client/src/index.css]: I suggest
animation: flash 1s forwards ease normal;

File: client/src/index.css (hunk 1 above)
Comment (VassilIordanov) [reply client/src/index.css]: Maybe this is less "in your face"
animation-iteration-count: 3;

Comment (evdhoudt) [reply client/src/index.css]: I'd suggest flashing 3 times (feels like it grabs enough attention), it could become annoying when it flashes 5 times.

**Pull request merged**